addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -143,15 +143,18 @@
                     || (state.propertyDefinition.comodel &amp;&amp; state.propertyDefinition.comodel.length))"
                     class="o_field_property_definition_value">
                     <td class="o_td_label align-top pe-2">
-                        <b t-if="state.propertyDefinition.type === 'boolean'">
-                            Default State
-                        </b>
-                        <b t-else="">
-                            Default Value
-                        </b>
+                        <label for="property-default">
+                            <b t-if="state.propertyDefinition.type === 'boolean'">
+                                Default State
+                            </b>
+                            <b t-else="">
+                                Default Value
+                            </b>
+                        </label>
                     </td>
                     <td>
                         <PropertyValue
+                            id="'property-default'"
```

Comment:
Isn't it risky to have this kind of ID? As it is generic, is there a way we could end up with multiple times the same ID in the form view?

Reply:
if we can manage to have 2 definition open yes, changed to an uuid